test/stdlib/Accelerate.swift

```diff
@@ -230,7 +230,7 @@ if #available(iOS 9999, macOS 9999, tvOS 9999, watchOS 9999, *) {
         vectorB = [5]
         scalarA = 2
         
-        vDSP.multiply(vectorSubtraction: (vectorA, vectorB),
+        vDSP.multiply(subtraction: (vectorA, vectorB),
```

Comment:
Why's it OK to have `subtraction:(a, b)` as a parameter name but we feel like we need a label for `subtract(a, by: b, result: c)` or `(b, from: a, result: c)` as a function? The asymmetry is weird, and these should be consistent.

Reply:
I chose a verb as the method name and a noun for the parameter name, but happy to change both to verbs. 

WRT `subtract`, see below, I think `subtract(_ lhs, _ rhs)` (for all operations) makes sense.

Reply:
OK, I've removed the `by` and `from` names and correctly order the subtract arguments. I've left out the `lhs` and `rhs` parameter names; IMHO having the pseudocode variable match the parameter name makes for easier reading. @stephentyrone, I still have the function name as verb, parameter as noun - let me know your thoughts.